### PR TITLE
replace license

### LINKED ISSUE #1501: What licence?
All the PHP source files say they're AGPL3+ but `LICENSE` is MIT.

```php
/**
 * Copyright (c) 1998-2014 Browser Capabilities Project
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Affero General Public License as
 * published by the Free Software Foundation, either version 3 of the
 * License, or (at your option) any later version.
 *
 * Refer to the LICENSE file distributed with this package.
 *
 * @category   Browscap
 * @copyright  1998-2014 Browser Capabilities Project
 * @license    MIT
 */
```

[(Source)](https://github.com/browscap/browscap/blob/7bfccd5de4b16f9300fa2524d195cd69c02f0af5/src/Browscap/Browscap.php#L2-L15)

### PR DESCRIPTION
**Pull request by mimmi20**
I replaced the license text in the files.

This fixes #1501.

### CODE REVIEW

File: src/Browscap/Browscap.php [hunk 1]
```diff
@@ -1,16 +1,27 @@
 <?php
 /**
- * Copyright (c) 1998-2014 Browser Capabilities Project
+ * Copyright (c) 1998-2017 Browser Capabilities Project
  *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
+ * Permission is hereby granted, free of charge, to any person obtaining a
```
Comment (asgrim) [inline src/Browscap/Browscap.php]: I don't think we need the whole license in the headers here. Just `Refer to the LICENSE file distributed with this package.` should suffice please.

File: src/Browscap/Browscap.php [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
  *
  * @category   Browscap
- * @copyright  1998-2014 Browser Capabilities Project
+ * @copyright  1998-2017 Browser Capabilities Project
```
Comment (asgrim) [inline src/Browscap/Browscap.php]: Remove `@category`, `@copyright` (or at least just have one copyright line, there's two atm) and `@license`, they're just unnecessary IMO

Comment (mimmi20) [inline src/Browscap/Browscap.php]: done

File: src/Browscap/Browscap.php (hunk 1 above)
Comment (mimmi20) [inline src/Browscap/Browscap.php]: done

**Comment by asgrim** [comment]
Thanks @mimmi20!

**Pull request merged**